paper-toast.html

```diff
@@ -187,6 +187,7 @@
          * Called when the value of `opened` changes.
          */
         _openedChanged: function() {
+          this.cancelDebouncer('auto-close');
```

Comment:
If we always cancel the debouncer, does it need to be `debounce`? Or should it be `async`?